ZConfig/cmdline.py

```diff
@@ -142,7 +174,10 @@ def _normalize_case(self, string):
         return string.lower()
 
 
-class MatcherMixin:
+class MatcherMixin: # pylint:disable=old-style-class
+    # Can't extend object without getting __init__ errors due to
```

Comment:
Probably? I did try the obvious trivial version (`super(MatcherMixin, self).__init__(*args)`) and that didn't work so I didn't look into it any further. But I can.